src/main/java/au/com/sealink/quicktravel/client/models/PassengerType.java

```diff
@@ -0,0 +1,141 @@
+package au.com.sealink.quicktravel.client.models;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Date;
+
+public class PassengerType {
+    @SerializedName("id")
+    @Expose
+    private Integer id;
+    @SerializedName("name")
+    @Expose
+    private String name;
+    @SerializedName("minimum_age")
+    @Expose
+    private Object minimumAge;
+    @SerializedName("maximum_age")
+    @Expose
+    private Object maximumAge;
```

Comment:
Why is defaultAge a Integer but minimum/maximumAge Object?

Reply:
Good pickup, the original code for these models was generated by a tool where these where no values for certain fields, so it defaults it back to Object